src/renderer/shared/ui/StatusLabel/common/types.ts

```diff
@@ -1 +1 @@
-export type Variant = 'waiting' | 'success' | 'warn' | 'error';
+export type Variant = 'waiting' | 'success' | 'warn' | 'error' | 'staking';
```

Comment:
I have not found any `<StatusLabel variant="staking">`
Moreover primitive ui component cannot know anything from our domain, in this example - `staking`

Reply:
We use `Variant` in the DotStyles type as well

Reply:
Taking new comments into account, `staking` must be deleted